src/google/protobuf/compiler/cpp/cpp_bootstrap_unittest.cc

```diff
@@ -178,6 +178,18 @@ TEST(BootstrapTest, GeneratedFilesMatch) {
   }
 }
 
+//test Generate in cpp_generator.cc
+TEST(BootstrapTest, optionNotExist) {
+  cpp::CppGenerator generator;
+  DescriptorPool pool_;
```

Comment:
Based on our [style guide](https://google.github.io/styleguide/cppguide.html#Variable_Names) we only put an underscore at the end of a variable name if it's a class data member, so `pool_` and `parameter_` should not have the underscore.

Reply:
fixed
And thank you for providing the style guide :)